lib/arbre/rails/template_handler.rb

```diff
@@ -1,3 +1,21 @@
+# frozen_string_literal: true
+
+ActionView::Base.class_eval do
```

Comment:
Perhaps this should be moved to its own file, named something like `lib/arbre/rails/capture.rb`